packages/contracts/lib.esm/index.d.ts

```diff
@@ -18,23 +19,21 @@ export interface PayableOverrides extends Overrides {
     value?: BigNumberish | Promise<BigNumberish>;
 }
 export interface CallOverrides extends PayableOverrides {
-    blockTag?: BlockTag | Promise<BlockTag>;
     from?: string | Promise<string>;
 }
 export interface PopulatedTransaction {
-    to?: string;
-    from?: string;
-    nonce?: number;
+    to?: AccountLike;
+    from?: AccountLike;
     gasLimit?: BigNumber;
-    gasPrice?: BigNumber;
-    data?: string;
-    value?: BigNumber;
+    data?: BytesLike;
```

Comment:
Lets keep the same interface for `data` to be `string`. What is the reason behind changing it?

Reply:
thought it's better to be more extendable but obviously, it's not a good idea in that case, gonna stick to the etherjs as far as possible in the future, my bad